### PR TITLE
Lots of things

### PR DESCRIPTION
**Pull request by emalsidog**
I have only one question about this part

image

I have temporarily commented your code to demonstrate that formatted messages are working. But i think we need different types for messages. To add html type, if it has html tags. Also I thing we need to fix sever response when getting messages. They have different structure and unnecessary fields (When getting searched results)

### CODE REVIEW

File: client/src/translates/ru.js [hunk 1]
```diff
@@ -77,6 +77,44 @@ export const ru = {
...
+  Pending: 'Ожидающее',
+  Closed: 'Закрытое',
+  'Search through all messages': 'Поиск по всем сообщениям',
+  'Search here': 'Ищите тут',
+  'Nothing found': 'Ничего не найдено',
+  'No files were found': 'Файлы не были найдены',
+  'No links were found': 'Ссылки не были найдены',
+  'No files or links were found': 'Ссылки или файлы не были найдены',
+  'Files and Links': 'Файлы и Ссылки',
+  Filter: 'Фильтер',
```
Comment (bubuzavrik) [inline client/src/translates/ru.js:94]: Фильтр

File: client/src/translates/ru.js [hunk 2]
```diff
@@ -77,6 +77,44 @@ export const ru = {
...
+  Files: 'Файлы',
+  Links: 'Ссылки',
+  People: 'Люди',
+  'Add People +': 'Добавить Людей +',
+  'Add People': 'Добавить Людей',
```
Comment (bubuzavrik) [inline client/src/translates/ru.js:99]: 98 and 99 rows same, better to use `${translateText('Add People')} +` in code and "+" you can add if needed

File: client/src/translates/ru.js [hunk 3]
```diff
@@ -77,6 +77,44 @@ export const ru = {
...
+  'Do not show chat history': 'Скрыть историю чата',
+  'Show all chat history': 'Показать всю историю чата',
+  Cancel: 'Отмена',
+  Add: 'Добавить',
+  'Write a message...': 'Напишите сообщение...',
+  "It's empty here": 'Здесь пусто',
+  Send: 'Отправить',
+  'Search in this room': 'Поиск в этом чате',
+  'Not assigned': 'Не поручено',
```
Comment (bubuzavrik) [inline client/src/translates/ru.js:108]: 108 and 118 row, already exists. debug:true also said about same translation keys

**Review by bubuzavrik — CHANGES_REQUESTED** [review]

File: client/src/translates/ru.js (hunk 3 above)
Comment (emalsidog) [reply client/src/translates/ru.js:108]: Hey, I have a question about deleting user. Can we delete SuperAdmin from chat or not? I have deleted him from the chat but server still gives me info about chats where user does not exist

File: client/src/components/Messages/Thread/index.js [hunk 4]
```diff
@@ -164,17 +191,26 @@ export const Thread = ({
...
-              onChange={handleMessageChange}
-            />
+        <form onSubmit={handleSendMessage}>
+          <CKEditor
+            editor={ClassicEditor}
+            data={message}
+            config={{
+              placeholder: translateText('Write a message...')
+            }}
+            onChange={(event, editor) => {
```
Comment (bubuzavrik) [inline client/src/components/Messages/Thread/index.js:201]: don't create callbacks in render

**Review by bubuzavrik — CHANGES_REQUESTED** [review]

**Pull request merged**